### PR TITLE
Add scim documentation

### PR DESCRIPTION
**Pull request by Mumakil**

### CODE REVIEW

File: docs/SCIM.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+The Single Sign-On for Flowdock Enterprise offers partial support for provisioning using [System for Cross-Domain Identity Management (SCIM)](http://www.simplecloud.info/). To use the SCIM api, you need to be a Flowdock Enterprise customer and have the Single Sign-On configured. If you haven't completed these steps, please contact [support@flowdock.com](mailto:support@flowdock.com).
+
+## Basics
+
+The SCIM api is a REST based api that can be found under `https://api.flowdock.com/scim/:uid`. Uid is an organization specific identifier that can be acquired when setting up the SSO. All the resources can be found under that path.
+
+The api uses JSON encoded data, so all requests need to be made using `application/json` content type and the data received back will be in the same format.
+
+## Autentication
```
Comment (anttipitkanen) [inline docs/SCIM.md]: Typo

File: docs/SCIM.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+### Search index (find by email)
+
+The Users index accepts an optional filter parameter, that can be used to search for users. Currently only supported filter is `userName eq "email@example.com"`, which allows searching for users using their email address. The response is in the same format as with index.
+
+#### Parameters
+
+| Parameter| Description |
+|----------|-------------|
+| **filter** | A scim filter string. Only the `userName eq` filter is supported at the moment. |
```
Comment (anttipitkanen) [inline docs/SCIM.md]: `:uid` could be listed here as in the other resources

**Comment by anttipitkanen** [comment]
Besides the few notes, looking good :ship:

**Comment by Mumakil** [comment]
I'm doing some tweaks, so don't merge yet.

**Comment by Mumakil** [comment]
Done.

**Pull request merged**